build_tools/packaging/python/templates/rocm/src/rocm_sdk/tests/core_test.py

```diff
@@ -37,13 +37,15 @@
     ("amdlld", ["-flavor", "ld.lld", "--help"], "USAGE:", True),
     ("hipcc", ["--help"], "clang LLVM compiler", True),
     ("hipconfig", [], "HIP version:", True),
+    ("hipify-clang", ["--help"], "AMD LLVM verison", True),
 ]
 
 LINUX_CONSOLE_SCRIPT_TESTS = [
     ("amd-smi", [], "AMD-SMI", True),
     ("rocm_agent_enumerator", [], "", True),
     ("rocminfo", [], "", True),
     ("rocm-smi", [], "Management", True),
+    ("hipify-perl", ["--help"], "HIP Version", True),
```

Comment:
Same here

```suggestion
    ("hipify-perl", ["--help"], "USAGE:", True),
```

Here's some of the output to consider:
```
    hipify-perl is a tool to translate CUDA source code into portable HIP C++

    USAGE: hipify-perl [OPTIONS] INPUT_FILE

    OPTIONS:

      -cuda-kernel-execution-syntax - Keep CUDA kernel launch syntax (default)
      -examine                      - Combines -no-output and -print-stats options
      -exclude-dirs=s               - Exclude directories
```